applications/nrf5340_audio/broadcast_source/main.c

```diff
@@ -367,6 +367,18 @@ static int ext_adv_populate(uint8_t big_index, struct broadcast_source_ext_adv_d
 	int ret;
 	size_t ext_adv_buf_cnt = 0;
 
+	if (IS_ENABLED(CONFIG_BT_AUDIO_USE_BROADCAST_NAME_ALT)) {
```

Comment:
This needs to check `brdcst_name_size` against `BROADCAST_SOURCE_ADV_NAME_MAX` (or have a build assert for it)